packages/grafana-ui/src/components/Table/TableNG/TableNG.tsx

```diff
@@ -427,20 +446,28 @@ export function TableNG(props: TableNGProps) {
             };
           }
 
+          const tooltipFieldName = field.config.custom?.tooltip?.field;
+          let ref: React.RefObject<HTMLDivElement> | undefined;
+          if (tooltipFieldName) {
+            ref = createRef();
+            cellRefsMatrix[rowIdx] = cellRefsMatrix[rowIdx] || [];
+            cellRefsMatrix[rowIdx][i] = ref;
+          }
```

Comment:
this is kind of unfortunate, but I can't figure out a better way to provide the popoverRef of the actual table cell to the TableCellTooltip.